sources/api/pluto/Cargo.toml

```diff
@@ -11,6 +11,7 @@ exclude = ["README.md"]
 
 [dependencies]
 bytes = "1"
+constants = { path = "../../constants", version = "0.1" }
```

Comment:
Where is my dollar? :money_mouth_face: 

https://github.com/bottlerocket-os/bottlerocket/pull/2238#issuecomment-1163980161